alert/store.go

```diff
@@ -791,3 +791,41 @@ func (s *Store) State(ctx context.Context, alertIDs []int) ([]State, error) {
 
 	return list, nil
 }
+
+func (s *Store) Feedback(ctx context.Context, alertID int) (f Feedback, err error) {
```

Comment:
Why check if we allow anything?

Reply:
It validates that it has some sort of authentication (e.g., they must be logged in, vs. open to the public). Unauthenticated calls will be rejected by `Any` since they don't have any type of access.